elastic/apm-server
# Extend ingest API to accept info about target service

## Motivation/summary
Allow ingest API to capture `service.target.name` and  `service.target.type`

## Checklist


## How to test these changes


1. Send spans with service.target.name and service.target.type set to non empty values
2. Observe the generated span documents to contain `span.service.target.name` and `span.service.target.type` values

## Related issues
#7715
